realm/src/main/java/io/realm/internal/UncheckedRow.java

```diff
@@ -283,4 +288,5 @@ public boolean isAttached() {
     protected native void nativeNullifyLink(long nativeRowPtr, long columnIndex);
     protected static native void nativeClose(long nativeRowPtr);
     protected native boolean nativeIsAttached(long nativeRowPtr);
+    protected native boolean nativeHasField(long nativeRowPtr, String columnName);
```

Comment:
Can be name be `nativeHasColumn`? It is consistent with the other column functions for example `nativeGetColumnCount()`.